
    public CertificateException(Throwable cause) {
        super(cause);
    }
}
